docker/action.yml

```diff
@@ -0,0 +1,55 @@
+name: "quarto-docker"
+description: "Build and push a Quarto Docker container to the GitHub Container Registry."
+inputs:
+  source:
+    description: "Full path to quarto file"
+    required: true
+  version:
+    description: "The version to tag the image with."
+    required: true
+  token:
+    description: "The token to authenticate to docker registry"
+    required: true
+  username:
+    description: "The username to authenticate to docker registry"
+    required: true
+  org:
+    description: 'The GitHub orginization that will host Docker image'
+    required: true
+  name:
+    description: 'The name of the Docker image'
+    required: true
+  daily:
+    description: "Apply the `daily` tag to the image."
+    required: false
+    default: true
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: echo "BASE_NM=ghcr.io/${{ inputs.org }}/${{ inputs.name }}" >> $GITHUB_ENV
+
+    - name: Create DockerFile
+      shell: bash
+      run: |
+        cat << 'EOF' >> Dockerfile
+        FROM ubuntu:22.04
+        COPY ${{ inputs.source }} quarto-linux-amd64.deb
+        RUN dpkg -i quarto-linux-amd64.deb && rm quarto-linux-amd64.deb
+        EOF
+
+    - if: ${{ inputs.daily != 'true' && inputs.daily != 'True'  }}
+      shell: bash
+      run: echo "LATEST_TAG=-t ${{ env.BASE_NM }}:latest" >> $GITHUB_ENV
+
+    - if: ${{ inputs.daily && inputs.daily != 'false' && inputs.daily != 'False' }}
+      shell: bash
+      run: echo "DAILY_TAG=-t ${{ env.BASE_NM }}:daily" >> $GITHUB_ENV
```

Comment:
should the tag be `prerelease` instead of daily ? 
This is the term we use - https://quarto.org/docs/download/prerelease.html and we can build several time per day really. 

Currently not a good example as we switched to latest stable vs RC.

Reply:
@jjallaire wanted the term to be `daily` [see slack](https://positpbc.slack.com/archives/C01LQETLA7L/p1679620438572129?thread_ts=1679602309.415929&cid=C01LQETLA7L)  I don't mind changing it though if there is a diff opinion.

Reply:
Let me paste his comment in here, that might be easier:

```
You probably want latest, pre-release-latest, and maybe daily ? or perhaps just latest and daily (as pre-release is just a web facing designation we give when the daily is stabilizing)
```

Reply:
Perfect ! Let it be daily then. Thanks for reminding me the context !